recipes/coffea/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set name = "coffea" %}
+{% set version = "0.6.44" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/coffea-{{ version }}.tar.gz
+  sha256: 9fa8c924f6c372cd32459648d30e8f383a3bd897c2ca6b31d695b25397a140aa
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - flake8
+    - pip
+    - pytest-runner
+    - python
+  run:
+    - awkward >=0.12.20
+    - awkward1 >=0.2.32
+    - cloudpickle >=1.2.3
+    - futures  # [py==27]
```

Conda-forge doesn't support Python 2 any more so this can be remove so the package can be `noarch: python` (meaning there will be only one package build instead of one per platform per architecture per Python version).

```suggestion
    - futures
```

It's possible to manually re-add Python 3 support but it's probably not worth the hassle at this point, unless you have a good reason to?